docs/scripts/install_kiss.ps1

```diff
@@ -98,6 +100,7 @@ kubectl create secret generic kiss-secrets `
     --from-literal=VAC_OBJECTEN_TOKEN=$(VAC_OBJECTEN_TOKEN) `
     --from-literal=VAC_OBJECTTYPES_TOKEN=$(VAC_OBJECTTYPES_TOKEN) `
     --from-literal=ZAAKSYSTEEM__0__API_KEY=$(ZAAKSYSTEEM__0__API_KEY) `
+    --from-literal=MANAGEMENTINFORMATIE_API_KEY=$(MANAGEMENTINFORMATIE_API_KEY) `
```

Comment:
Vraag: Bij een eerdere PR had Felix hier toegevoegd: "Deze staat al bij de secrets. Verder prima!"
Ik weet niet of dit betekent, dat deze regel eigenlijk weg kan?

Reply:
daar staat hij nu ook weer. was blijkbaar weggevallen